### PR TITLE
docs: describe preset options

### PR DESCRIPTION
**Pull request by zyyv**

### CODE REVIEW

File: docs/presets/wind.md [hunk 1]
```diff
@@ -48,11 +48,6 @@ import { presetWind } from 'unocss'
 ```
 :::
 
-## Rules
```
Comment (antfu) [inline docs/presets/wind.md:51]: Can we keep this?

**Review by antfu — COMMENTED** [review]
I guess you need to enable corepack or upgrade your pnpm. The lock is diffing

File: docs/presets/uno.md [hunk 2]
```diff
@@ -60,7 +61,10 @@ For example, both `ml-3` (Tailwind), `ms-2` (Bootstrap), `ma4` (Tachyons), and `
 .mt-10px { margin-top: 10px; }
 ```
 
-## Rules
```
[the remark this answers is not in the document]
Comment (zyyv) [reply docs/presets/uno.md:63]: I'm sorry, I'm based on an old version.😅

**Comment by zyyv** [comment]
> I guess you need to enable corepack or upgrade your pnpm. The lock is diffing

In fact, that is the lock file after I upgraded.😅  
Now, I have downgraded the pnpm version same as `packageManager`.

File: docs/presets/icons.md [hunk 3]
```diff
@@ -298,6 +294,101 @@ presetIcons({
...
+:::
+
+### unit
+
+- Type: `string`
+- Default: `'em'`
+
+Custom icon unit.
+
+### cdn
```
Comment (userquin) [inline docs/presets/icons.md:380]: I need to review current state for runtime, maybe we need to add some more hints here

Comment (userquin) [reply docs/presets/icons.md:380]: the PR LGTM, can be added later if necessary

Comment (userquin) [reply docs/presets/icons.md:380]: https://github.com/unocss/unocss/pull/1555

**Pull request merged**